server/app/services/databaseConnection/DBConnection.java

```diff
@@ -0,0 +1,109 @@
+package services.databaseConnection;
+
+import play.Configuration;
+import play.api.Play;
+
+import java.sql.*;
+import java.util.ArrayList;
+
+/**
+ * Created by sabinapokhrel on 11/7/16.
+ */
+
+
+public class DBConnection {
+  private Configuration configuration = Play.current().injector().instanceOf(Configuration.class);
+  /**
+   * Method to connect to database
+   * @return
+   */
+  Connection connectDB() {
+
+    /* These variables must be made configurable */
+    String host = configuration.getString("db-conf.host");
+    String dbName = configuration.getString("db-conf.dbName");
+    String username = configuration.getString("db-conf.username");
+    String password = configuration.getString("db-conf.password");
+
+    try {
+      /* Dynamically load the JDBC driver */
+      Class.forName("org.postgresql.Driver"); //load the driver
+
+      /* Connect to the Postgres database */
+      Connection db = DriverManager.getConnection(
+        "jdbc:postgresql://" + host + "/" + dbName,
+        username,
+        password); //connect to the db
+      return db;
+    } catch (Exception e) {
+      e.printStackTrace();
+    }
+    return null;
+  }
+
+  /**
+   * SQL statement to add the question to database
+   * @param question
+   */
+  public void addQuestionToDB(String question) {
+    DBConnection DBConnection = new DBConnection();
+    Connection connection = DBConnection.connectDB();
+
+    DBConnection.createDatabase(connection);
+
+    try{ try {
```

Comment:
@haolinj The first try is for the close connection inside finally. Removing it gives an error. Please let me know if there is a better way to implement it.